apps/prairielearn/src/pages/instructorAssessmentRegrading/instructorAssessmentRegrading.html.ts

```diff
@@ -0,0 +1,159 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+import { z } from 'zod';
+
+import { JobSequenceSchema, UserSchema } from '../../lib/db-types';
+import { Modal } from '../../components/Modal.html';
+
+export const RegradingJobSequenceSchema = z.object({
+  job_sequence: JobSequenceSchema,
+  start_date_formatted: z.string(),
+  user_uid: UserSchema.shape.uid,
+});
+type RegradingJobSequence = z.infer<typeof RegradingJobSequenceSchema>;
+
+export function InstructorAssessmentRegrading({
+  resLocals,
+  regradingJobSequences,
+}: {
+  resLocals: Record<string, any>;
+  regradingJobSequences: RegradingJobSequence[];
+}) {
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(__filename, "<%- include('../partials/head'); %>", resLocals)}
+      </head>
+      <body>
+        <script>
+          $(function () {
+            $('[data-toggle="popover"]').popover({ sanitize: false });
+          });
+        </script>
+        ${renderEjs(__filename, "<%- include('../partials/navbar'); %>", resLocals)}
+        <main id="content" class="container-fluid">
+          ${renderEjs(
+            __filename,
+            "<%- include('../partials/assessmentSyncErrorsAndWarnings'); %>",
+            resLocals,
+          )}
+          ${resLocals.authz_data.has_course_instance_permission_edit
+            ? html`
+                ${regradeAllAssessmentInstancesModal({
+                  assessmentSetName: resLocals.assessment_set.name,
+                  assessmentNumber: resLocals.assessment.number,
+                  csrfToken: resLocals.__csrf_token,
+                })}
+              `
+            : ''}
+
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white">
+              ${resLocals.assessment_set.name} ${resLocals.assessment.number}: Regrading
+            </div>
+
+            ${resLocals.authz_data.has_course_instance_permission_edit
+              ? html`
+                  <div class="card-body">
+                    <p>
+                      <button
+                        type="button"
+                        class="btn btn-primary"
+                        data-toggle="modal"
+                        data-target="#regrade-all-form"
+                      >
+                        <i class="fa fa-sync" aria-hidden="true"></i>
+                        Regrade all assessment instances
+                      </button>
+                    </p>
+                    <p class="small">
+                      To regrade assessment instances for specific students, use the "Action"
+                      dropdown on the
+                      <a
+                        href="${resLocals.urlPrefix}/assessment/${resLocals.assessment
+                          .id}/instances"
+                        >assessment instances page </a
+                      >.
```

Comment:
Here's a point where you want the space to be removed instead at the end. Not sure how Prettier will handle this.
```suggestion
                      <a
                        href="${resLocals.urlPrefix}/assessment/${resLocals.assessment
                          .id}/instances"
                        > assessment instances page</a
                      >.
```